### PR TITLE
Update documentations to new format

### PR DESCRIPTION
**Pull request by kapilraajP**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: gpeople/Module.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        refreshToken: <REFRESH_TOKEN>
+    }
+};
+
+people:Client googleContactClient = check new (googleContactConfig);
+```
+
+### Step 3: Invoke connector operation
+
+1. You can create a contact as follows with `createContact` method by passing a `Person` record and `FieldMask[]` as parameters. Successful creation returns the created contact as a `PersonResponse` and the error cases returns an `error` object.
```
Comment (RukshiW) [inline gpeople/Module.md:41]: Suggestion: "Create a contact via the `createContact method by passing a `Person` record and `FieldMask[]` as parameters."

Remove the second sentence about returns.

File: gpeople/Module.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            unstructuredName: "Jason Hardy"
+        }]
+    };
+    people:FieldMask[] personFields = [people:NAME, people:PHONE_NUMBER, people:EMAIL_ADDRESS];
+    people:PersonResponse createContact = check googleContactClient->createContact(person, personFields);
+    ```
+2. Use `bal run` command to compile and run the Ballerina program. 
+
+## Quick reference
+The following code snippets shows how the connector operations can be used in different scenarios after initializing the client.
```
Comment (RukshiW) [inline gpeople/Module.md]: Change "shows" to "show". End the sentence with a colon because it precedes a list.

File: gpeople/Package.md [hunk 3]
```diff
@@ -1,320 +1,21 @@
-# Ballerina Google People Connector 
-Connects to Google People using Ballerina.
+Connects to Google People API from Ballerina
```
Comment (RukshiW) [inline gpeople/Package.md:1]: Should "Ballerina" be linked to the Ballerina site?

File: gpeople/Package.md [hunk 4]
```diff
@@ -1,320 +1,21 @@
...
-# Introduction
-## Google People
-[Google People](https://developers.google.com/people) is a contact-management service developed by Google. It lets users to organize their schedule and share events with others. The Google People endpoint allows you to access the Google People API Version v1 through Ballerina.
+## Package overview
+The `ballerinax/googleapis.people` is a [Ballerina](https://ballerina.io/) connector for Google People API.
 
-## Key Features of Google People
-* Manage Contacts
-* Manage Contact Groups
+This package provides the capability to access Google People API service.
```
Comment (RukshiW) [inline gpeople/Package.md:6]: Add the article "the" before "Google People API service".

File: gpeople/Package.md (hunk 3 above)
Comment (kapilraajP) [reply gpeople/Package.md:1]: As it is found across other connectors in similar way i added that link

File: gpeople/Module.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
+## Overview
+Ballerina connector for Google People API is connecting the Google People API via Ballerina language easily. It provides capability to perform operations related to contacts and contact groups in Google Contacts.
```
[the remark this answers is not in the document]
Comment (kapilraajP) [reply gpeople/Module.md]: Fixed

File: gpeople/Module.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+This module supports [Google People API](https://developers.google.com/people) V1.0 version.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create [Google Account](https://accounts.google.com/signup/v2/webcreateaccount?hl=en&flowName=GlifWebSignIn&flowEntry=SignUp)
```
[the remark this answers is not in the document]
Comment (kapilraajP) [reply gpeople/Module.md]: Fixed

File: gpeople/Module.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+* Obtaining tokens
```
[the remark this answers is not in the document]
Comment (kapilraajP) [reply gpeople/Module.md]: Fixed

File: gpeople/Module.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+To use the Google People API connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector
+First, import the ballerinax/googleapis.people module into the Ballerina project.
+```ballerina
+import ballerinax/googleapis.people as people;
+```
+### Step 2: Create a new connector instance
+
+You can now enter the credentials in the Google People API client configuration and create Google People API client by passing the configuration:
```
[the remark this answers is not in the document]
Comment (kapilraajP) [reply gpeople/Module.md]: Fixed

File: gpeople/Module.md (hunk 1 above)
Comment (kapilraajP) [reply gpeople/Module.md]: Fixed

File: gpeople/Module.md (hunk 2 above)
Comment (kapilraajP) [reply gpeople/Module.md]: Fixed

**Review by abeykoon — APPROVED** [review]

**Pull request merged**